src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Data/TodoTemplateDbContext.cs

```diff
@@ -19,6 +19,59 @@ protected override void OnModelCreating(ModelBuilder builder)
         ConfigIdentityTables(builder);
     }
 
+    #region Override Save Changes
+
+    public override int SaveChanges()
+    {
+        try
+        {
+            return base.SaveChanges();
+        }
+        catch (DbUpdateConcurrencyException exception)
+        {
+            throw new ConflictException("Database concurrency exception happens", exception);
```

Comment:
Use `ErrorStrings` and `nameof` instead

Reply:
Could you please propose a meaningful error message here? @msynk

Reply:
how about something like this:

`The record was modified by another user after you got the original data. the operation was canceled.`

Reply:
@milad-karimifard Please move this to ErrorStrings and use that via `nameof`